README.md

```diff
@@ -535,6 +535,41 @@ No additional changes are required to run the pipeline except updating the `--fi
 ./gradlew run ... -Pargs="... --filePattern=gs://${PROJECT_ID}-demo-data/*.parquet"
 ```
 
+#### 6. ORC
+
+##### Inspection of ORC files
+The inspection pipeline will read ORC files from input GCS storage bucket and the results of will be written in BigQuery
+tables.
+
+```commandline
+./gradlew run ... -Pargs="... --filePattern=gs://${PROJECT_ID}-demo-data/*.orc"
+```
+
+##### De-identification
+The de-identification pipeline support ORC file format where data can be read from input GCS storage bucket and the 
+de-identified results can be written in 
+1. output BigQuery dataset as tables, or 
```

Comment:
1. output BigQuery dataset as tables, or --> To write the de-identified files to a BigQuery table:
OR
To write the de-identified files to BigQuery tables

(depending on which one is more accurate)